schema.json

```diff
@@ -0,0 +1,660 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "StoryRAMP Config Schema",
+    "type": "object",
+    "additionalProperties": false,
+
+    "$defs": {
+        "slide": {
+            "type": "object",
+            "description": "A universal StoryRAMP slide",
+            "properties": {
+                "panel": {
+                    "$ref": "#/$defs/panel"
+                }
+            },
+            "required": ["panel"]
+        },
+
+        "panel": {

```

Comment:
we talked a bit about this but it might be good to allow the users to customize the width of a panel and otherwise default to a set ratio (don't think the default can be specified in schema currently tho with how it's setup so it would be determined in the code) depending on what panels make up a slide

Reply:
added a temporary width for now, but we can discuss how we want the final design to look